README.md

```diff
@@ -26,38 +29,39 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
 * [Unity](https://unity3d.com/): A cross-platform game engine used to develop video games for PC, consoles, mobile devices and websites.
 
 # Watch the Video
+
 [![](https://i.ytimg.com/vi/o-OQc-hHtoU/0.jpg)](https://youtu.be/o-OQc-hHtoU)
 
 # Steps
 
 1. [Before you begin](#1-before-you-begin)
-2. [Create Bluemix services](#2-create-bluemix-services)
+2. [Create IBM Cloud services](#2-create-ibm-cloud-services)
 3. [Building and Running](#3-building-and-running)
 
-## 1. Before You Begin
+### 1. Before You Begin
```

Comment:
Not sure why these should skip a level?

Reply:
That's the pattern from the [journey docs](https://github.ibm.com/developer-journeys/journey-docs/blob/master/_content/resources/templates/readme_template.md)
But missing are the `deploy locally` and `run on Bluemix` levels.
So, not sure whether to keep all the levels the same, even if missing the higher level, or have them decrement...

Reply:
I don't think it's going to break anything, unless you use a *really* strict markdown validator (I've seen similar stuff fail for RST). Either way work for me, just depending on how you intend the headers to be rendered.

Reply:
Yeah, I figured skip a level and they'll all have consistent size for the common steps.